howso/marginal.amlg

```diff
@@ -790,23 +770,23 @@
 								))
 							)
 
-							;output assoc of marginal stats for the feature, replace .nan with null if needed
+							;output assoc of marginal stats for the feature
 							(assoc
-								min (if (!= .nan min) min)
-								max (if (!= .nan max) max)
-								mean (if (!= .nan mean) mean)
-								median (if (!= .nan median) median)
-								percentile_25 (if (!= .nan percentile_25) percentile_25)
-								percentile_75 (if (!= .nan percentile_75) percentile_75)
-								mode (if (!= .nan mode) mode)
+								min min
```

Comment:
not saying you need do this, but this is a fitting place to use a { } style for this assoc, imo

Reply:
Don't really want to do a one-off here in this PR